app/assets/stylesheets/administrate/components/_form.scss

```diff
@@ -35,7 +37,13 @@
 
   [type="checkbox"],
   [type="radio"] {
-    margin: $small-spacing 0 ($base-spacing + $form-input-padding);
+    margin: $small-spacing 0 ($base-spacing + ($base-spacing / 3));
```

Comment:
`$base-spacing + ($base-spacing / 3)` should be written without spaces around the operator: `$base-spacing+$base-spacing / 3`<br>`$base-spacing / 3` should be written without spaces around the operator: `$base-spacing/3`